File: Android/Java/app/src/main/java/com/unity3d/levelplaydemo/DemoImpressionDataListener.java
package com.unity3d.levelplaydemo;

import static com.unity3d.levelplaydemo.DemoActivity.logCallbackName;

import androidx.annotation.NonNull;

import com.unity3d.mediation.impression.LevelPlayImpressionData;
import com.unity3d.mediation.impression.LevelPlayImpressionDataListener;

public class DemoImpressionDataListener implements LevelPlayImpressionDataListener {
    private final String TAG = DemoImpressionDataListener.class.getSimpleName();

    /**
     Called when the ad was displayed successfully and the impression data was recorded
     @param levelPlayImpressionData The recorded impression data
     */
    @Override
    public void onImpressionSuccess(@NonNull LevelPlayImpressionData levelPlayImpressionData) {
        logCallbackName(TAG, "");
    }
}

File: Android/Kotlin/app/src/main/java/com/levelplay/levelplaysdkdemo/DemoImpressionDataListener.kt
package com.levelplay.levelplaysdkdemo

import com.levelplay.levelplaysdkdemo.DemoActivity.Companion.logCallbackName
import com.unity3d.mediation.impression.LevelPlayImpressionData
import com.unity3d.mediation.impression.LevelPlayImpressionDataListener

class DemoImpressionDataListener : LevelPlayImpressionDataListener {
    private val TAG = DemoImpressionDataListener::class.java.name

    /**
    Called when the ad was displayed successfully and the impression data was recorded
    @param impressionData The recorded impression data
     */
    override fun onImpressionSuccess(impressionData: LevelPlayImpressionData) {
        logCallbackName(TAG, "")
    }
}